### PR TITLE
Fix defaultresourcegroup in CloudShell

### PR DESCRIPTION
**Pull request by maddieclayton**
## Description
https://github.com/Azure/azure-powershell/issues/5682

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#making-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
sign job: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/264/
on demand: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/629/ (and 630)

File: tools/AzureRM.Example.psm1 [hunk 1]
```diff
@@ -45,12 +45,15 @@ else
...
-$FilteredCommands | ForEach-Object {
-	$global:PSDefaultParameterValues.Add($_,
-		{
-			$context = Get-AzureRmContext
-			if (($context -ne $null) -and $context.ExtendedProperties.ContainsKey("Default Resource Group")) {
-				$context.ExtendedProperties["Default Resource Group"]
-			} 
-		})
-}
+if ($Env:ACC_ClOUD -eq $null)
```
Comment (markcowl) [inline tools/AzureRM.Example.psm1]: consistent casing?

File: src/ResourceManager/Profile/Commands.Profile/Default/GetAzureRmDefault.cs [hunk 2]
```diff
@@ -37,6 +37,11 @@ public class GetAzureRMDefaultCommand : AzureRMCmdlet
 
         public override void ExecuteCmdlet()
         {
+            if (Environment.GetEnvironmentVariable("ACC_CLOUD") != null)
```
Comment (markcowl) [inline src/ResourceManager/Profile/Commands.Profile/Default/GetAzureRmDefault.cs]: remove

**Comment by markcowl** [comment]
on demand run here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/634/

**Pull request merged**